packages/web/src/scss/components/Tooltip/index.html

```diff
@@ -413,19 +419,20 @@ <h2 class="docs-Heading">Dismissible Tooltip via JS API and Floating UI</h2>
       <script type="module" src="dismissible-tooltip.mjs"></script>
 
       <div class="TooltipWrapper d-inline-block" data-spirit-element="tooltip-wrapper">
-        <button id="my-dismissible-button" class="Button Button--primary Button--medium" aria-describedby="my-dismissible-tooltip-with-floating-ui">I have a tooltip 😎</button>
+        <button id="my-dismissible-button" class="Button Button--primary Button--medium" data-spirit-target="#my-dismissible-tooltip-with-floating-ui" aria-describedby="my-dismissible-tooltip-with-floating-ui">I have a tooltip 😎</button>
         <div
           id="my-dismissible-tooltip-with-floating-ui"
           class="Tooltip Tooltip--dismissible is-hidden"
           data-spirit-placement="right"
           data-spirit-flip="true"
           data-spirit-flip-cross-axis="true"
           data-spirit-flip-fallback-placements="right, top-end"
+          data-spirit-trigger="asdas"
```

Comment:
```suggestion
          data-spirit-trigger="manual"
```

We know this will work but please use rather `manual` for better readability and understanding of what it does.

Reply:
fixed